### PR TITLE
Remove 'export' keyword where not needed.

### PR DESCRIPTION
**Pull request by Shazib**
As discussed in #1126 creating separate PRs.

The linter rule isn't actually part of this first PR and will likely be in the final PR as otherwise it will just fail because all the issues aren't resolved. 

The code is 'ready' though so as soon as this is merged I will open next one(s)

### CODE REVIEW

File: packages/desktop-client/src/components/FixedSizeList.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function ResizeObserver({ onResize, children }) {
   return children(ref);
 }
 
-export class FixedSizeList extends PureComponent {
+export default class FixedSizeList extends PureComponent {
```
Comment (j-f1) [inline packages/desktop-client/src/components/FixedSizeList.js:18]: Have you noticed a convention for using vs not using default exports? (is there a way to have a lint rule for that?)

**Review by j-f1 — COMMENTED** [review]
All changes seem fine to me but at some point I think we should address using vs not using default exports.

File: packages/desktop-client/src/components/FixedSizeList.js (hunk 1 above)
Comment (Shazib) [reply packages/desktop-client/src/components/FixedSizeList.js:18]: I can look into if there is a linter rule. Its certainly not 100% but it felt like 'default' was used more often than not so I elected to make that change.

Example from a random page, only a few of those don't have a default export e.g. `ManagePayeesPage`

```
import Account from './accounts/Account';
import MobileAccount from './accounts/MobileAccount';
import MobileAccounts from './accounts/MobileAccounts';
import { ActiveLocationProvider } from './ActiveLocation';
import BankSyncStatus from './BankSyncStatus';
import Budget from './budget';
import { BudgetMonthCountProvider } from './budget/BudgetMonthCountContext';
import MobileBudget from './budget/MobileBudget';
import { View } from './common';
import FloatableSidebar, { SidebarProvider } from './FloatableSidebar';
import GlobalKeys from './GlobalKeys';
import { ManageRulesPage } from './ManageRulesPage';
import Modals from './Modals';
import NordigenLink from './nordigen/NordigenLink';
import Notifications from './Notifications';
import { PageTypeProvider } from './Page';
import { ManagePayeesPage } from './payees/ManagePayeesPage';
import Reports from './reports';
import Schedules from './schedules';
import DiscoverSchedules from './schedules/DiscoverSchedules';
import EditSchedule from './schedules/EditSchedule';
import LinkSchedule from './schedules/LinkSchedule';
import PostsOfflineNotification from './schedules/PostsOfflineNotification';
import Settings from './settings';
import Titlebar, { TitlebarProvider } from './Titlebar';
```

**Review by j-f1 — APPROVED** [review]
Makes sense, thanks for that context!

**Pull request merged**